### ISSUE TITLE
Undisplayable Bitcoin Content

### ISSUE THREAD

**Issue by Elshaffah** [GeekMagicClock/smalltv-pro#51]
Hi GeekMagic, I just got the smallTV pro and after connecting it to my home router/wifi and finally entered the Bitcoin section, it does not display contents on this section
![image](https://github.com/GeekMagicClock/smalltv-pro/assets/66177516/23b9bbb2-acae-48e0-b932-90c3fb220404)

**Comment by GeekMagicClock** [comment]
Hi, is the internet connected?

**Comment by Elshaffah** [comment]
Yes my internet is connected and my smallTV pro is also connected to the same internet router/wifi

**Comment by GeekMagicClock** [comment]
Is the binance.com or binance.us accessble in your network?

**Comment by Elshaffah** [comment]
Yes I can access it with my network

**Comment by GeekMagicClock** [comment]
Is the other info correct? Like weather, stocks ?

**Comment by DadRe-dmt** [comment]
> Yes I can access it with my network So what could be this cause of these infos not displaying and how do I rectify it?

Did you enter the coinpairs (set coin) in the correct syntax ? btcusdt ltcusdt

And make sure the pairs you enter are available at Binance. 

Using incorrect syntax and/or enter coinpairs that are not available at Binance will result in no charts being displayed.
